### DISCUSSION TITLE
Noncommercial licenses may be incompatible with modrinth

### DISCUSSION THREAD

**Discussion by SoniEx2** [modrinth/.github#37] [Other discussions]
Since there's no way to turn off ads and revenue sharing for a project, it's possible modrinth may be entirely incompatible with noncommercial licenses as used in e.g. mod assets.

We know we wouldn't wanna upload our forks of other mods to modrinth in the off-chance we get sued.

**Comment by osfanbuff63** [comment]
You kinda can turn off earning revenue by giving yourself a 0% share. I haven't done that myself, but I'm sure there's someone here who has.

**Comment by triphora** [comment]
The exact wording used by the CC licenses is "primarily intended for or directed toward commercial advantage or monetary compensation". I would interpret this to mean activities related to buying and selling rather than simply having a small bit of advertising.
